executor.go

```diff
@@ -302,7 +533,15 @@ func (e *Executor) runPass() {
 					var duration time.Duration
 					var err error
 
-					if task.Run != nil {
+					if task.RunWithFlags != nil {
```

Comment:
Just double checking, do we panic if somebody registers both RunWithFlags & Run?

Reply:
No panic, we could do that though 👍

Reply:
Done (in registry commit)